trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>can you also include here some typical fields may be included such as app_id?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Added this to `TruVirtual` and added more to the docstring visible from noted help(TruVirtual).